pkg/apis/serving/v1beta1/revision_validation_test.go

```diff
@@ -92,31 +92,35 @@ func TestContainerConcurrencyValidation(t *testing.T) {
 		want *apis.FieldError
 	}{{
 		name: "single",
-		cc:   1,
+		cc:   ptr.Int64(1),
 		want: nil,
 	}, {
 		name: "unlimited",
-		cc:   0,
+		cc:   ptr.Int64(0),
 		want: nil,
 	}, {
 		name: "ten",
-		cc:   10,
+		cc:   ptr.Int64(10),
 		want: nil,
 	}, {
 		name: "invalid container concurrency (too small)",
-		cc:   -1,
+		cc:   ptr.Int64(-1),
 		want: apis.ErrOutOfBoundsValue(-1, 0, RevisionContainerConcurrencyMax,
 			apis.CurrentField),
 	}, {
 		name: "invalid container concurrency (too large)",
-		cc:   RevisionContainerConcurrencyMax + 1,
+		cc:   ptr.Int64(RevisionContainerConcurrencyMax + 1),
 		want: apis.ErrOutOfBoundsValue(RevisionContainerConcurrencyMax+1,
 			0, RevisionContainerConcurrencyMax, apis.CurrentField),
 	}}
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			got := test.cc.Validate(context.Background())
+			var got *apis.FieldError
+			if *test.cc < 0 || *test.cc > RevisionContainerConcurrencyMax {
+				got = apis.ErrOutOfBoundsValue(
+					*test.cc, 0, RevisionContainerConcurrencyMax, apis.CurrentField)
+			}
```

Comment:
I don't like this, the logic will diverge and this isn't testing any real code (just a point in time copy of it).

Can you fold this into the testing for the enclosing object?